Makefile

```diff
@@ -55,16 +55,17 @@ OUT_IMAGE = $(OUT_IMAGE_NAME):$(OUT_IMAGE_TAG)
 
 ##### Public rules #####
 DISTRIBUTIONS := ubuntu18.04 ubuntu20.04 ubuntu22.04 ubuntu24.04 signed_ubuntu20.04 signed_ubuntu22.04 signed_ubuntu24.04 rhel8 rhel9 flatcar fedora36 sles15.3 precompiled_rhcos
+RHCOS_VERSIONS := rhcos4.14 rhcos4.15 rhcos4.16 rhcos4.17 rhcos4.18 rhel9.6
```

Comment:
Actually, the more that I think about it, it's better to remove this line. During the build stage, we just need a vgpu manager image tag ending with `-rhel9`. That image can be retagged as necessary to support the desired RHCOS/RHEL versions. That's the process we follow

Reply:
```suggestion
```

Reply:
Discussed offline. Resolving